ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -124,6 +124,13 @@ public int execute() {
     try {
       long loadTaskStartTime = System.currentTimeMillis();
       SecurityUtils.reloginExpiringKeytabUser();
+      //Don't proceed if target db is replication incompatible.
+      if (work.dbNameToLoadIn != null) {
```

Comment:
When would this be null?

Reply:
we can have some situation in which work.dbNameToLoadIn is not set.

Reply:
Do we know when could that be?

Reply:
we have some cases which to test REPL LOAD * scenarios.